### ISSUE TITLE
Update Git clone command

### ISSUE THREAD

**Issue by Aggror** [stride3d/stride-website#150]
Reminder for myself to make a PR for this
![image](https://github.com/stride3d/stride-website/assets/3499539/ede5b9e2-0576-4f5d-b1c8-9e1ce2e2d199)

**Comment by VaclavElias** [comment]
Yes, @manio143 mentioned that as well. We have to update it across all references and repos.
